src/model/guild/role.rs

```diff
@@ -307,7 +307,7 @@ impl FromStrAndCache for Role {
 ///
 /// [Discord docs](https://discord.com/developers/docs/topics/permissions#role-object-role-tags-structure).
 #[derive(Clone, Debug, Default, Deserialize, Serialize)]
-#[cfg_attr(test, derive(PartialEq))]
+#[cfg_attr(test, derive(PartialEq, Eq))]
```

Comment:
```suggestion
#[cfg_attr(test, derive(Eq, PartialEq))]
```
Keep these in alphabetical order.

Reply:
I agree with the goal of having consistency, however, serenity doesn't sort derives in lots of places

![Screenshot_20220927_154207](https://user-images.githubusercontent.com/21220820/192542871-4a7786aa-e726-420d-bd25-cecab983d01d.png)

For consistency, all should be fixed, but in a separate PR. But even then, I think it's futile to manually order derives. The slight aesthetic satisfaction is not worth the laborious editing.

Eventually rustfmt should get support to automatically sort derives